manifests/ros-image-demo/deployment.yaml

```diff
@@ -3,6 +3,8 @@ toolchainRegion: us-west-2
 groups:
   - name: optionals
     path: manifests/ros-image-demo/optional-modules.yaml
+  - name: viz
+    path: manifests/ros-image-demo/visualization-modules.yaml
```

Comment:
Is it a good idea to launch this instance in a separate group before core?  It would be adding 25+ minutes to the deploy time

Reply:
Agreed. Will move it to end of manifest